ops-tools/docker/images/app/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM maven:alpine
```

Comment:
Could you please specify which maven:alpine version you want to use? You already did it for postgres and redis. Please do it for maven also. It's one of the best practices - always specify version of images.